pysparkling/rdd.py

```diff
@@ -23,17 +23,20 @@
 except ImportError:
     numpy = None
 
-from . import fileio
-from .exceptions import FileAlreadyExistsException, ContextIsLockedException
-from .samplers import (BernoulliSampler, PoissonSampler,
-                       BernoulliSamplerPerKey, PoissonSamplerPerKey)
-from .stat_counter import StatCounter
+from pysparkling import fileio
```

Comment:
Any reason you are changing the imports? Explicit relative imports are preferred.

Reply:
Hmm, I ran into an issue while using relative imports but as I am unable to reproduce it I'll undo the change,

On the other hand in some of the additional code it seems to have a bad effect on readability:

Relative: https://github.com/tools4origins/pysparkling/blob/withRelativeImport/pysparkling/sql/internal_utils/readers/csvreader.py
vs
Absolute: https://github.com/tools4origins/pysparkling/blob/bb57b521e4d89e109f43430b94460de0d7bb6a4f/pysparkling/sql/internal_utils/readers/csvreader.py

Reply:
Indeed. Is that `internal_utils` structure coming from the Scala implementation? That level of nesting seems very Java and would be nice to avoid in Python. In any case, thanks for moving to relative imports here and lets discuss the sql part in another PR.

Reply:
It is indeed replicating Scala separation of most of the logic from the API itself, I agree that another PR is the right time to speak about it